packages/generate-cnpj/src/cnpj.js

```diff
@@ -0,0 +1,26 @@
+export const CNPJ_LENGTH = 14;
+
+export const FIRST_DIGIT_WEIGHTS = [5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2];
+
+export const SECOND_DIGIT_WEIGHTS = [6, 5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2];
+
+export const generateRandomNumbers = n =>
+  Array(n)
+    .fill(0)
+    .map(() =>
+      Math.random()
+        .toString()
+        .substr(2, 1)
+    );
+
+export const multiplyWeights = (numbers, weights) => numbers.map((number, index) => number * weights[index]);
+
+export const sumNumbers = numbers => numbers.reduce((sum, number) => sum + number);
+
+export const calculateNthDigit = sum => {
```

Comment:
You could use https://github.com/brazilian-utils/brazilian-utils/blob/master/packages/helper-generate-checksum to solve that

Reply:
That function was not used because I was concerned about readability. I though its name might hide the logic of the algorithm, but it was changed in f75b51d75186874d0e03f88a3c803ba7c2c81e67 to favor code reuse.

Let me know if it's good enough. :+1: